packages/@react-spectrum/tooltip/stories/TooltipTrigger.stories.tsx

```diff
@@ -100,6 +100,94 @@ storiesOf('TooltipTrigger', module)
         <Button variant="secondary">No Tooltip</Button>
       </Flex>
     )
+  )
+  .add(
+    'crossoffset examples',
```

Comment:
I number of these seem positioned wrong in Chrome OSX. I attached to images, but more do these two cases for me.
![image](https://user-images.githubusercontent.com/309540/95524058-2b7f5e80-0985-11eb-91e1-7a2dd79e54a4.png)
![image](https://user-images.githubusercontent.com/309540/95524080-3b973e00-0985-11eb-9bf1-c17c4802b597.png)

Reply:
yeah, i made massive offsets to illustrate that it wouldn't leave, but arrow size is unknown by the overlay, so i think devon's thought was this is fine, we'll trust people not to go bananas